src/validation/accommodationSchema.js

```diff
@@ -65,6 +65,11 @@ const accomodationFeedbackSchema = Joi.object({
   message: JoiValidator.validateString().required(),
 });
 
+const accommodationRatingSchema = Joi.object({
+  accommodationId: isValidId,
+  value: JoiValidator.validateNumber().min(1).max(5).required()
```

Comment:
add validation for integer here. currently the app throw an error if decimal is supplied

Reply:
Okay.. thanks for pointing that out.

Reply:
Done!